JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -128,20 +135,45 @@ fun ArticleScreen(
         topBar = {
             TopAppBar(
                 title = {
-                    Text(
-                        text = "Published in: ${post.publication?.name}",
-                        style = MaterialTheme.typography.subtitle2,
-                        color = LocalContentColor.current
-                    )
+                    Column(modifier = Modifier.padding(top = 10.dp, bottom = 5.dp)) {
```

Comment:
This column looks redundant

Reply:
I removed the column and assign its padding values to the row instead